### PR TITLE
FloorsAndRoomCard: Change creation of title for LightCard

### PR DESCRIPTION
**Pull request by Rosi2143**
uiSemantics is based on points of equipments to this point is used to create the title instead of the equipment itself

Signed-off-by: Rosi2143 <Schrott.Micha@web.de>

### CODE REVIEW

File: mainWidget/main_widget_FloorsAndRooms.yaml [hunk 1]
```diff
@@ -51,8 +51,9 @@ slots:
                           default:
                             - component: widget:main_widget_Light_Card
                               config:
-                                Title: "=loop.lightItem.metadata.uiSemantics ? loop.lightItem.metadata.uiSemantics.config.equipment + loop.lightItem.metadata.uiSemantics.config.preposition + loop.lightItem.metadata.uiSemantics.config.location : loop.lightItem.label"
+                                Title: "=loop.lightPowerItem.metadata.uiSemantics ? loop.lightPowerItem.metadata.uiSemantics.config.equipment + loop.lightPowerItem.metadata.uiSemantics.config.preposition + loop.lightPowerItem.metadata.uiSemantics.config.location : loop.lightItem.label"
```
Comment (hmerk) [inline mainWidget/main_widget_FloorsAndRooms.yaml:54]: Why changing from the equipment Item to the switch item ?

File: mainWidget/main_widget_FloorsAndRooms.yaml [hunk 2]
```diff
@@ -51,8 +51,9 @@ slots:
...
                                 groupItem: =loop.lightItem.name
+                                groupCategory: =loop.lightItem.category
                                 powerItem: = loop.lightPowerItem.name
```
Comment (hmerk) [inline mainWidget/main_widget_FloorsAndRooms.yaml:56]: Why introducing an new param to pass to the LightCard ?

Comment (Rosi2143) [reply mainWidget/main_widget_FloorsAndRooms.yaml:56]: Os sorry that was in intermediate solution only.

At the end it turned out that the title just needed to be changed to make it work.

**Comment by hmerk** [comment]
Still not answered my first question.

**Comment by Rosi2143** [comment]
Hi,

sorry missed the first question.

In my setup the equipment is not part of another equipment - and iI suppose this is a general rule.
So the uiSemantic of the equipment does not exist at all, or at least `loop.lightItem.metadata.uiSemantics.config.equipment` is empty. This makes the title something like
"im Wohnzimmer" instead of "Licht im Wohnzimmer".

As we have the item already - with full uiSemantic - this seems to be the more appropriate solution

It removes an implicit rule that an equipment must have a `loop.lightItem.metadata.uiSemantics.config.equipment` pointing to itself.

**Comment by hmerk** [comment]
There is no rule to use uiSemantics at all. If not used, the label of the equipment is considered.
This should also work if for some reason not having a switch in the group.
I need to think about that topic a bit more, as I see several different use cases, but for now, I would like to not implement that change and close this PR.
For not to forget this topic, I will open an issue for this change request.

**Comment by hmerk** [comment]
Superseeded by #39

**Pull request closed without merging**